Parent/Parent/Course/CourseDetailsViewController.swift

```diff
@@ -90,6 +91,24 @@ class CourseDetailsViewController: HorizontalMenuViewController {
         viewControllers.append(vc)
     }
 
+    func configureComposeMessageButton() {
+        guard ExperimentalFeature.parentInbox.isEnabled else { return }
+        let buttonSize: CGFloat = 56
+        let margin: CGFloat = 16
+        let bottomMargin: CGFloat = 50
+
+        replyButton = FloatingButton(frame: CGRect(x: 0, y: 0, width: buttonSize, height: buttonSize))
+        replyButton?.iconName = "replySolid"
+        replyButton?.iconColorName = "white"
```

Comment:
I would prefer calling `setImage` & setting the `tintColor` directly instead of using the `@IBInspectable` stringly-typed values in code.